<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WebDialog</class>
 <widget class="QDialog" name="WebDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>362</width>
    <height>208</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>From Web...</string>
  </property>
  <property name="windowIcon">
   <iconset theme="web-browser">
    <normaloff>.</normaloff>.</iconset>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="labelRA">
     <property name="text">
      <string>R. A.</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="editRA"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="labelDec">
     <property name="text">
      <string>Dec.</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="editDec"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="labelRadius">
     <property name="text">
      <string>Radius</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QSpinBox" name="spinBoxRadius">
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>15</number>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QWidget" name="widget" native="true"/>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="2">
    <widget class="QProgressBar" name="progressBar">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="value">
      <number>0</number>
     </property>
     <property name="textVisible">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
